src/commands/account/add.ts

```diff
@@ -0,0 +1,194 @@
+  static description = "add an account";
+
+  static flags = {
+    ...Command.flags,
+    name: flags.string({ char: "n", description: "account name" }),
+    email: flags.string({ char: "e", description: "your email" }),
+    password: flags.string({ char: "p", description: "your password" }),
+  };
+
+  async run() {
+    const { flags } = this.parse(AccountAdd);
+    const debug = createDebugLogger(flags.debug);
+    const liara_json: ILiaraJson = this.gatherLiaraJson();
+    const pervAccounts = this.gatherOtherAccounts();
+    const name = flags.name ? flags.name : await this.promptName();
+    const region = flags.region ? flags.region : await this.promptRegion();
+    if (!flags.email) {
+      let emailIsValid = false;
+      do {
+        flags.email = await this.promptEmail();
+        emailIsValid = validateEmail(flags.email);
+        if (!emailIsValid) {
+          process.stdout.write(eraseLines(1));
+        }
+      } while (!emailIsValid);
+
+      this.log();
+    }
+    const body = {
+      email: flags.email,
+      password: flags.password ? flags.password : await this.promptPassword(),
+    };
+
+    this.axiosConfig.baseURL = REGIONS_API_URL[region];
+
+    const { api_token } = (await retry(
+      async () => {
+        try {
+          const { data } = await axios.post(
+            "/v1/login",
+            body,
+            this.axiosConfig
+          );
+          return data;
+        } catch (err) {
+          debug("retrying...");
+          throw err;
+        }
+      },
+      { retries: 3 }
+    )) as { api_token: string };
+
+    const accounts = {
+      ...pervAccounts,
+      [name]: {
+        email: body.email,
+        api_token,
+        region,
+      },
+    };
+
+    fs.writeFileSync(
+      GLOBAL_CONF_PATH,
+      JSON.stringify({
+        api_token: liara_json.api_token,
+        region: liara_json.region,
+        current: liara_json.current,
+        accounts,
+      })
+    );
+
+    this.log(`> Auth credentials saved in ${chalk.bold(GLOBAL_CONF_PATH)}`);
+    this.log(`> Current account is: ${liara_json.current}`);
+  }
+
+  async promptRegion(): Promise<string> {
+    const { selectedRegion } = (await prompt({
+      name: "selectedRegion",
+      type: "list",
+      message: "Please select a region:",
+      choices: ["iran", "germany"],
+    })) as { selectedRegion: string };
+
+    return selectedRegion;
+  }
+
+  async promptName(): Promise<string> {
+    const { name } = (await prompt({
+      name: "name",
+      type: "input",
+      message: "enter your prefered name:",
+      validate(input) {
+        if (input.length === 0) {
+          return false;
+        } else {
+          return true;
+        }
+      },
+    })) as { name: string };
+    const pervAccounts = await this.gatherOtherAccounts();
+    const pervAccountsName = pervAccounts && Object.keys(pervAccounts);
+    return pervAccountsName?.includes(name)
+      ? this.error("this name in used for another account")
```

Comment:
`This name has already been used for another account. Please use a different name.